x-pack/platform/plugins/shared/task_manager/server/integration_tests/task_priority_check.test.ts

```diff
@@ -55,6 +55,11 @@ describe('Task priority checks', () => {
         !!taskType.priority ? { taskType: taskType.type, priority: taskType.priority } : null
       )
       .filter((tt: { taskType: string; priority: TaskPriority } | null) => null != tt);
+
+    await retry(async () => {
```

Comment:
should it retry the entire process of getting the task definitions from the dictionary? otherwise we're just retrying the length check on a fixed array